spmd/__init__.py

```diff
@@ -1,6 +1,5 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates
-from typing import Optional, Callable, Tuple, Union
-import torch
+from typing import Optional, Callable
```

Comment:
just being curious, why `Tuple` and `Union` were imported even it wasn't used before?

Reply:
idk tbh.. maybe the import lint check ignoring `__int__.py`